app/src/main/java/com/github/polypoly/app/utils/Constants.kt

```diff
@@ -0,0 +1,44 @@
+package com.github.polypoly.app.utils
+
+import androidx.compose.ui.unit.dp
+
+/**
+ * Constants used in the app
+ */
+class Constants {
+
+    companion object {
+        /**
+         * Constants for the code generator
+         */
+        const val GAME_LOBBY_CODE_LENGTH = 5
```

Comment:
Are all these constants used in a different class than `CreateGameLobbyActivity`? If not, they shouldn't be here.

Reply:
This one yes (will be used in `JoinGameLobby `to set a max character to code textfield). Anyway for any other constants, they may or may not be used someday elsewhere. But even in the case that they are only used in `CreateGameLobbyActivity`, I think having all important values ("constants") for our code not dispatched everywhere is a good idea, to easely monitor those in one place. I think it is subject to debate. Maybe we can create specific Constants files in packages to have a better abstraction, like GameLobbyConstants or else... For now I will let these here and I suggest we decide something about it!